#choose between inverse Kineatics and Inverse Dynamics
IK: "kinematics"
#IK : "dynamics"

# Parameter for invers kinematics

InversKinematics: {
  damp : 0.001,
  alphaKin : 0.1,
  gamma : 0.5,
  margin : 0.07,
  tolerance : 0.5,
}

# Parameter for inverse velocities
InversDynamics: {
  alphaVel : 0.8,
  proportional_gain : 1.0,
  linear_velocity_limit : 1.0 ,
  angular_velocity_limit : 1.0,
  }